### DISCUSSION TITLE
No Icon or Title displayed within Dialog

### DISCUSSION THREAD

**Discussion by omersuve** [phantom/docs#325] [Help & Support] [answered]
### Summary

I couldnt solve my issue of showing icon and title when user interacts my app

I added apple-touch-icon, and normal icon as well
I added meta tags with open graph

Still couldnt manage it

### Example

https://shotbots.app/

### Steps to Reproduce

could you check the head tag of the source html?

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
@omersuve Please provide a screenshot of where in the wallet you are referring to.

**Comment by omersuve** [comment, marked as answer]
It is resolved, issue was about my middleware that blocks icon files.

Thank you for responding!

Adam ***@***.***>, 3 Ara 2024 Sal, 03:44 tarihinde şunu yazdı:
